custom_components/givenergy_local/sensor.py

```
@@ -490,20 +490,43 @@ class GivEnergyCoordinatorSensorDescription(SensorEntityDescription):
         value_fn=lambda inv: inv.p_load_demand,
     ),
     GivEnergyInverterSensorDescription(
-        key="e_load_day",
-        name="Load Energy Today",
+        # Derived house consumption (single-phase only) — matches the GE app's
+        # "Consumption today". Single-phase inverters expose no consumption
+        # register; givenergy-modbus computes it (PV gen + grid-in − grid-out −
```

Ruff (0.15.15)

[warning] Comment contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)